rest/rest-cli/src/main/resources/org/ow2/proactive_grid_cloud_portal/cli/cmd/commands.js

```diff
@@ -249,16 +251,53 @@ function listjobs(x, y) {
     }
 }
 
+// expected params: jobId, tag, offset, limit 
+function listtasks(arg1, arg2, arg3, arg4) {
```

Comment:
Why not replacing the variables with the expected parameter names? or parameter names with a prefix or suffix if name is already used?